.github/CODEOWNERS

```diff
@@ -29,9 +29,6 @@ SYCL/DotProduct @rdeodhar
 # Explicit SIMD
 SYCL/ESIMD @kbobrovs @DenisBakhvalov
 
-# Function pointers
-SYCL/FunctionPointers @AlexeySachkov
```

Comment:
Who is going to be code owner for the directory?

Reply:
No one. The directory is removed. Deprecated tests are owned by llvm-revievers-runtime.